src/core/templates/buttons.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="buttons-display">
         <div class="buttons-marking-icon icon display-none">
         </div>
-        <div class="buttons-display-inner">
+        <div class="buttons-display-inner" aria-live="rude">
```

Comment:
oh I know there is a "component complete, correct" but I meant something like an aria-live which reads it out automatically as there isn't really a guarantee that the user tabs down that far. Just a thought.